easybuild/easyconfigs/c/cutadapt/cutadapt-1.16-foss-2017b-Python-3.6.3.eb

```diff
@@ -19,16 +19,26 @@ description = """Cutadapt finds and removes adapter sequences, primers, poly-A t
 
 toolchain = {'name': 'foss', 'version': '2017b'}
 
-source_urls = [PYPI_SOURCE]
-sources = [SOURCELOWER_TAR_GZ]
-checksums = ['9432045dcf59802ef9bb2d1f7232e8a015eb9b984e7c7ff35a6c8a57681e7a79']
-
 dependencies = [
     ('Python', '3.6.3'),
 ]
 
 use_pip = True
-download_dep_fail = True
+
+exts_list = [
+    ('bz2file', '0.98', {
```

Comment:
@vanzod I think this one is only needed for Python 2.7.x (see also last commit in #7562)